<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-09T18:37:04 IST" name="Default suite" finished-at="2022-08-09T18:37:08 IST" duration-ms="3599">
    <groups>
    </groups>
    <test started-at="2022-08-09T18:37:04 IST" name="Default test" finished-at="2022-08-09T18:37:08 IST" duration-ms="3599">
      <class name="dataDrivenpack.PostAPIWithMultipleDataSet">
        <test-method signature="signUPwithDataDriven(java.lang.Object,java.lang.Object,java.lang.Object)[pri:0, instance:dataDrivenpack.PostAPIWithMultipleDataSet@3835c46]" started-at="2022-08-09T18:37:05 IST" name="signUPwithDataDriven" data-provider="signUpData1" finished-at="2022-08-09T18:37:07 IST" duration-ms="1458" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[07ggabhi@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[gaurone]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUPwithDataDriven -->
        <test-method signature="signUPwithDataDriven(java.lang.Object,java.lang.Object,java.lang.Object)[pri:0, instance:dataDrivenpack.PostAPIWithMultipleDataSet@3835c46]" started-at="2022-08-09T18:37:07 IST" name="signUPwithDataDriven" data-provider="signUpData1" finished-at="2022-08-09T18:37:07 IST" duration-ms="410" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[08ggabhi@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[gaurtwo]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[212121]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUPwithDataDriven -->
        <test-method signature="signUPwithDataDriven(java.lang.Object,java.lang.Object,java.lang.Object)[pri:0, instance:dataDrivenpack.PostAPIWithMultipleDataSet@3835c46]" started-at="2022-08-09T18:37:07 IST" name="signUPwithDataDriven" data-provider="signUpData1" finished-at="2022-08-09T18:37:07 IST" duration-ms="404" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[09ggabhi@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[gaurthree]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[313131]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUPwithDataDriven -->
        <test-method signature="signUPwithDataDriven(java.lang.Object,java.lang.Object,java.lang.Object)[pri:0, instance:dataDrivenpack.PostAPIWithMultipleDataSet@3835c46]" started-at="2022-08-09T18:37:07 IST" name="signUPwithDataDriven" data-provider="signUpData1" finished-at="2022-08-09T18:37:08 IST" duration-ms="368" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at dataDrivenpack.PostAPIWithMultipleDataSet.signUPwithDataDriven(PostAPIWithMultipleDataSet.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUPwithDataDriven -->
      </class> <!-- dataDrivenpack.PostAPIWithMultipleDataSet -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
